templates/kubernetes/config.yml

```diff
@@ -1,5 +1,5 @@
 name: Kubernetes
 description: |
   Manage a cluster of Linux containers as a single system to accelerate Dev and simplify Ops
-version: v1.1.0-rancher1
+version: v1.2.0-rancher1
```

Comment:
The 1.1.0 is intended to match the k8s version, so this should be 1.1.0-rancher2